04/app.js

```diff
@@ -6,4 +6,16 @@ const user = {
         month: '04',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+const currentDate = new Date(1985, 4, 14);
```

Comment:
Taki zapis oznacza, że będzie 14 maja 1985. Natomiast użytkownik ma urodziny w kwietniu.
Trzeba na to uważać - zapis "ludzki" różni się od "komputerowego" ;)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth